front-end/src/ducks/action.js

```diff
@@ -1,29 +1,42 @@
-const {UPDATE_NAME, GET_USERS, GET_PROFILE_DETAILS, UPDATE_COLOR} = require('./actionType')
-const {getUsers, getProfile} = require('./utils/api')
-const {yellow} = require('../components/Themes/themes')
+const type = require('./actionType')
```

Comment:
isn't a huge issue but typically this would be imported and named something like `ActionTypes`

Reply:
So we would name it `ActionTypes` and then access `ActionTypes.whatever`?